source/audioDucking.py

```diff
@@ -94,8 +103,14 @@ def setAudioDuckingMode(mode):
 	global _audioDuckingMode, _modeChangeEvent
 	if not isAudioDuckingSupported():
 		raise RuntimeError("audio ducking not supported")
-	if mode<0 or mode>=len(audioDuckingModes):
-		raise ValueError("%s is not an audio ducking mode")
+	# Normalize mode to within the exceptable range of values
+	mode=max(min(mode,len(audioDuckingModes)-1),0)

```

Comment:
I assume you do this so that old configurations won't break if the user set it to always duck? A comment here would be nice as to why we normalise instead of throwing an error.